libs/CCAppCommon/src/ccApplicationBase.cpp

```diff
@@ -273,15 +273,41 @@ void ccApplicationBase::setupPaths()
 
 bool ccApplicationBase::setAppStyle(QString styleKey)
 {
-	QStyle* style = QStyleFactory::create(styleKey);
-	if (!style)
-	{
-		ccLog::Warning("Invalid style key or style couldn't be created: " + styleKey);
-		return false;
-	}
+	const auto loadStyleSheet = [this](const QString& ressourcePath) {
```

Comment:
It's to force me to apply the clang format to the whole project? 😆 

For now can you manually move all these '{' to the next line?

Reply:
:wink: